packages/commonwealth/server/migrations/20240502120431-reconcile-xrefs.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface) => {
+    await queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.sequelize.query(
+        `
+-- delete orphan reactions
+delete from public."Reactions" where 
+not exists(select address_id from public."Addresses" where id=address_id);
+
+alter table only public."Reactions" 
+add constraint "Reactions_address_id_fkey"
+foreign key (address_id) references public."Addresses"(id);
+
+-- Many-to-many (X-ref) pattern with composite PK
+
+ALTER TABLE ONLY public."ThreadSubscriptions" DROP COLUMN IF EXISTS "id";
+ALTER TABLE ONLY public."ThreadSubscriptions"
+ADD CONSTRAINT "Threadsubscriptions_pkey" PRIMARY KEY (thread_id, user_id);
```

Comment:
Might as well remove `id` from `CommunityAlerts` too...

Reply:
Merging master 😱 ... removing snapshot tables and making community alerts an x-ref in next commit.